### PR TITLE
Ft#167720715 Design reset password form via email.

### PR DESCRIPTION
**Pull request by eliezerbasubi**
#### What does this PR do?
Adds reset password user experience functionalities to the UI. User can reset his password with the provided email.

#### Description of Task to be completed?
~Design reset password form
~Style form
~Validate email
~Display snackbar on success

#### How should this be manually tested?
The UI template can be found on gh-pages link.

#### Any background context you want to provide?
`N/A`
#### What are the relevant pivotal tracker stories?
#167720715

### CODE REVIEW

File: UI/css/stylesheet.css [hunk 1]
```diff
@@ -1049,6 +1060,23 @@ header .navbar {
...
+}
+.border-radius{
+    border-radius: 10px;
+}
+#login-form button{
+    margin-top: 1.1rem;
+    background: #000;
+}
+
+/* NEW FEATURE, RESET PASSWORD */
```
Comment (Kasulejoseph) [inline UI/css/stylesheet.css]: Refactor this file, Create different CSS files for all Html files that perform different functionalities.
For example the index page and the login page should have different CSS files.

**Review by Kasulejoseph — CHANGES_REQUESTED** [review]

**Comment by Kasulejoseph** [comment]
Refactor the dashboard file as well. Let the dashboard file only have dashboard content nothing else.

**Comment by Kasulejoseph** [comment]
Is this a backend PR or frontend? AM really confused.

File: UI/css/stylesheet.css (hunk 1 above)
Comment (eliezerbasubi) [reply UI/css/stylesheet.css]: Thank you i am going to do it right now.

**Pull request merged**